keps/0024-parameter-enhancement.md

```diff
@@ -0,0 +1,146 @@
+---
+kep-number: 24
```

Comment:
This should be `draft-YYYYMMDD` according to the [KEP process doc](https://github.com/kudobuilder/kudo/blob/master/keps/0001-kep-process.md#kep-metadata). :thinking:

Reply:
Oh wait, [not necessarily](https://github.com/kudobuilder/kudo/blob/master/keps/0001-kep-process.md#git-and-github-implementation) :man_facepalming: